pkg/webhook/fleetresourcehandler/fleetresourcehandler_webhook.go

```diff
@@ -24,11 +23,11 @@ import (
 
 const (
 	// ValidationPath is the webhook service path which admission requests are routed to for validating custom resource definition resources.
-	ValidationPath             = "/validate-fleetresourcehandler"
-	groupMatch                 = `^[^.]*\.(.*)`
-	fleetMemberNamespacePrefix = "fleet-member"
-	fleetNamespacePrefix       = "fleet"
-	kubeNamespacePrefix        = "kube"
+	ValidationPath                                = "/validate-fleetresourcehandler"
+	groupMatch                                    = `^[^.]*\.(.*)`
+	allowedMessageMemberCluster                   = "upstream member cluster resource is allowed to be created/deleted by any user"
```

Comment:
Hi Arvind! Just a nit: shall we put the format/rule strings and the messages into separate blocks?